ansible/idrsystems-playbooks/idr-analysis.yml

```diff
@@ -34,3 +34,9 @@
   - role: openmicroscopy.versioncontrol-utils
 
   - role: openmicroscopy.celery-docker
+
+- hosts: idr-analysis
+  tasks:
+  - file:
+      path: /scratch
+      mode: "1777"
```

Comment:
Could do, though up to now I've avoided deleting stuff in roles/playbooks unless necessary, on the grounds that you can never be sure whether that's equivalent to not having added something in the first place (e.g. installing packages automatically brings in deps), so you'd generally want to re-iamge before any big change in the role of a server.